docs/fuzzing.md

```diff
@@ -0,0 +1,44 @@
+### How
+
+Enter dev shell with fuzzing support:
+
+```
+nix develop .#fuzz
+```
+
+Select fuzzing target:
+
+```
+just fuzz-target
+```
+
+Starting fuzzing:
+
+```
+just fuzz-target <target>
+```
+
+If you have found a crash:
+
+```
+just fuzz-target-debug <target>
+```
+
+When `lldb` shows up:
```

Comment:
gdb has better rust support right?

Reply:
Not sure. I tried both and couldn't tell a difference.

It defaults to `rust-lldb` which lowers to lldb. https://github.com/rust-fuzz/honggfuzz-rs?tab=readme-ov-file#hfuzz_debugger